agreements/project-contribution/terms-and-conditions.md

```diff
@@ -0,0 +1,292 @@
+This period involves:
+
+- **Resource Assessment:**
+    - Inventory of all tangible and intangible assets, including code repositories, documentation, trademarks, and domain names.
+    - Evaluation of current financial status and funding sources.
+    - Identification of any outstanding liabilities or obligations.
+
+- **Project Boundaries:**
+    - Clear definition of the project’s scope, goals, and deliverables.
+    - Definition of project governance and leadership structures.
+
+The initial inventory of assets provided to the Sponsor is intended to provide a comprehensive list of resources necessary for the Project’s operations.
+This inventory is subject to periodic review and amendment as new assets are acquired or developed by the Project.
+Either the Sponsor or the Project may initiate updates to this inventory as needed, with all amendments documented in writing and appended to the original agreement.
+
+### Agreement Execution
+
+After the discovery period is complete and all details are thoroughly assessed and agreed upon, a [Fiscal Sponsorship Agreement][fs-landing] will be executed.
+
+#### Transfer of Assets
+
+The transfer of assets associated with the Project may involve one or more [Asset Transfer Agreements][fs-landing], depending on the scope and nature of the resources being transferred.
+Each agreement will specify the terms and inventory of assets to ensure a comprehensive and organized transition.
+
+These agreements will cover all assets essential for the Project’s continuity under the Sponsor’s management, as mutually agreed by the Project Signatories and the Sponsor.
+
+## Compliance and Governance
+
+### Adherence to Policies
+
+- **Code of Conduct:**
+Projects must maintain their own Code of Conduct for project activities.
+However, for all foundation-related activities, projects must adhere to the Sponsor’s [Code of Conduct][coc].
+
+- **Foundation Policies:**
+Projects must adhere to all Foundation Policies, including but not limited to these Terms, the [Trademark Policy][], and the [IP Policy][].
+
+- **Open Source Software:**
+Projects must use an open source license and ensure code is publicly available at all times.
+
+### Required Project Files
+
+Projects must provide and maintain the following documentation:
+
+- **CODE\_OF\_CONDUCT.md:**
+A document outlining the expected behavior and guidelines for project participants.
+
+- **CONTRIBUTING.md:**
+A document describing criteria for contributions as required by the [IP Policy][], versioning practices, and project build, test, and packaging instructions.
+
+- **GOVERNANCE.md:**
+A document describing project decision-making processes and contact information.
+
+Further details and templates for these files can be found in [Sponsored project guidelines][].
+
+### Communication
+
+Regular communication with the Sponsor regarding project activities and financial needs is required.
```

Comment:
What does this mean? Are we going to be required to write quarterly board reports 😭

Reply:
I don't want to read quarterly board reports either. Are you participating in EGC discussions? Are you around on Discord where I can find you?

We should be working together to make sure domains are renewed and any other expenses get paid. What we don't want is ghosting.